src/layouts/BlogLayout/BlogLayoutSidebar/BlogLayoutSidebar.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FunctionComponent } from 'react';
+import { groupBy } from 'lodash';
+import { Accordion, AccordionProps } from '@chakra-ui/react';
+
+import BlogPost from '@root/typings/blogPost';
```

Comment:
RC: `@typings/BlogPost` - fix import alias and file name with capitalize first letter